packages/cms-cli/commands/hubdb.js

```diff
@@ -1,198 +1,48 @@
-const path = require('path');
-const {
-  loadConfig,
-  validateConfig,
-  checkAndWarnGitInclusion,
-} = require('@hubspot/cms-lib');
-const { logger } = require('@hubspot/cms-lib/logger');
-const { logErrorInstance } = require('@hubspot/cms-lib/errorHandlers');
-const { getCwd } = require('@hubspot/cms-lib/path');
-const {
-  createHubDbTable,
-  downloadHubDbTable,
-  clearHubDbTableRows,
-} = require('@hubspot/cms-lib/hubdb');
-const { publishTable, deleteTable } = require('@hubspot/cms-lib/api/hubdb');
+const { addHelpUsageTracking } = require('../lib/usageTracking');
+const { addLoggerOptions } = require('../lib/commonOpts');
+const { configureCreate, CREATE_DESCRIPTION } = require('./hubdb/create');
+const { configureFetch, FETCH_DESCRIPTION } = require('./hubdb/fetch');
+const { configureClear, CLEAR_DESCRIPTION } = require('./hubdb/clear');
+const { configureDelete, DELETE_DESCRIPTION } = require('./hubdb/delete');
```

Comment:
Was there a reason for using the `configure*` approach instead of exporting a yargs command like we are doing elsewhere?

Reply:
The way I understood it is yargs needs you to export a single object containing those commands (command, describe, builder) so this approach seemed like the best way, since it builds up the whole of the commands within the one larger command.  

It also seemed a lot more clear and clean to me to introduce a folder-based structure for the subcommands with each of them separated.

Reply:
I was thinking that https://github.com/HubSpot/hubspot-cms-tools/blob/e2385f61bffa2d05fea619a557b08471aae68aca/packages/cms-cli/commands/secrets.js#L32-L35 was more straightforward.